src/main/java/ana_raetcaia/task_seven/ParseToIntException.java

```diff
@@ -0,0 +1,9 @@
+package ana_raetcaia.task_seven;
+
+public class ParseToIntException extends RuntimeException{
+    //Create personal Checked exception with name "ParseToIntException" and method which throws
+    // this excecption if a string provided as parameter can not be converted to int with Integer.parseToInt() method.
+    public  ParseToIntException (String input){
+        super(input);
```

Comment:
recommend to change error message to "Can not parse int: " + input.
otherwise you will receive just input string without details what happened.